### ISSUE TITLE
HomeKit Gerätetyp Speaker hinzufügen.

### ISSUE THREAD

**Issue by Jemeins** [rdmtc/RedMatic-HomeKit#141]
Ist es möglich den Gerätetyp Speaker in HomeKit mit dem Universal Accessory zu erstellen?

**Comment by hobbyquaker** [comment]
Klar, das geht. Aber Speaker Services werden weder von der home.app noch von Siri unterstützt. Mit Eve kann man sie nutzen.

**Comment by Jemeins** [comment]
Ich habe schon heutiger Bilder wie dieses gesehen.
https://i.redd.it/a3twoiubaf911.jpg
Gibt es die Darstellung denn nur auf Grund einer Beta?

**Comment by hobbyquaker** [comment]
das sind airplay2 speaker, das ist was komplett anderes als der homekit speaker service
